packages/connection/src/class-manager.php

```diff
@@ -1590,7 +1590,9 @@ public function get_token( $data ) {
 			),
 		);
 
+		add_filter( 'http_request_timeout', array( $this, 'increase_timeout' ) );
```

Comment:
Let's bump the priority on this filter to something high -- say `9999` or even `PHP_INT_MAX - 1`-- so we can still change it, but the average filter won't hit it.

Generally, we don't want other plugins to be modifying this without intent.

Reply:
Good point! I changed the priority to PHP_INT_MAX-1 in commit 5011a63.